examples/pysdl2.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python2
+
+#
+# Simple SDL2 / cefpython3 example.
+#
+# Only handles mouse events but could be extended to handle others.
+#
+# Requires pysdl2 (and SDL2 library).
+#
+# Tested under Fedora Linux (x86_64).
+#
+# by Neil Munday (www.mundayweb.com)
+#
+
+import os
+import sys
+import sdl2
+import sdl2.ext
+
+from cefpython3 import cefpython as cef
+from PIL import Image
+
+class LoadHandler(object):
+	def OnLoadingStateChange(self, browser, is_loading, **_):
+		if not is_loading:
+			print "loading complete"
+			
+	def OnLoadError(self, browser, frame, error_code, failed_url, **_):
+		if not frame.IsMain():
+			return
+		print "Failed to load %s" % failed_url
+		cef.PostTask(cef.TID_UI, exit_app, browser)
+
+class RenderHandler(object):
+	
```

Comment:
LoadHandler class doessn't have a blank line here, so this one shouldn't too to be consistent.